components/simplified-grid/grid.css

```diff
@@ -0,0 +1,36 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap");
+
+.grid-wrapper {
+  font-family: "Montserrat", sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.grid-container-2 {
+  display: grid;
+  border: 1px solid var(--txt-grey-600);
+  padding: 1rem;
+  width: 43.75rem;
+  gap: 1.25rem;
+  border-radius: 10px;
+  background-color: white;
+  grid-template-columns: auto auto;
+}
+.grid-item {
+  color: var(--txt-grey-100);
+  text-align: center;
+  padding: 10px;
```

Comment:
Refactor later: make sure to use consistent CSS units across all places, ( say majority of your code units are rem, so keep it rem instead of px)

resource: https://nekocalc.com/px-to-rem-converter